src/lib-components/GridGallery.vue

```diff
@@ -0,0 +1,230 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+
+import type { PropType } from 'vue'
+import type { GridGalleryItemType } from '@/types/types'
+
+import ResponsiveImage from '@/lib-components/ResponsiveImage.vue'
+import SmartLink from '@/lib-components/SmartLink.vue'
+
+const { featured, items } = defineProps({
+  items: {
+    type: Array as PropType<GridGalleryItemType[]>,
+    default: () => [],
+  },
+  featured: {
```

Comment:
I think this prop is not needed.

Reply:
@pghorpade It is used by the parsedAspectRatio computed property which passes its value to the ResponsiveImage component, and affects the size of the responsive images.